core/src/test/java/org/apache/iceberg/rest/responses/TestDropNamespaceResponse.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.rest.responses;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.apache.iceberg.AssertHelpers;
+import org.apache.iceberg.rest.RequestResponseTestBase;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TestDropNamespaceResponse extends RequestResponseTestBase<DropNamespaceResponse> {
+
+  @Test
+  public void testRoundTripSerDe() throws JsonProcessingException {
+    assertRoundTripSerializesEquallyFrom(
+        "{\"dropped\":true}", DropNamespaceResponse.builder().dropped(true).build());
+
+    assertRoundTripSerializesEquallyFrom(
+        "{\"dropped\":false}", DropNamespaceResponse.builder().dropped(false).build());
+  }
+
+  @Test
+  public void testFailParsingWhenNullOrEmptyJson() {
+    String nullJson = null;
+    AssertHelpers.assertThrows("DropNamespaceResponse should fail to deserialize from a null JSON string",
+        IllegalArgumentException.class,
+        () ->  deserialize(nullJson));
```

Comment:
Exception message validation?

Reply:
This exception message comes from Jackson and is `"argument "content" is null"`. I can add it in just so we know it came from Jackson, but it's not the most useful.

Reply:
Similarly, empty string `""` throws an exception from Jackson of `"No content to map due to end-of-input"`. I can add it in so it's clear it came from Jackson if we'd like.

Reply:
I added them in so it would be clear where they came from.